modules/topicsFpdModule.js

```diff
@@ -251,6 +260,25 @@ function loadTopicsForBidders() {
         setLoadedIframeURL(new URL(iframeURL).origin);
         iframeURL && window.document.documentElement.appendChild(ifrm);
       }
+
+      if (bidder && fetchUrl) {
+        let storedSegments = new Map(safeJSONParse(coreStorage.getDataFromLocalStorage(topicStorageName)));
+        const bidderLsEntry = storedSegments.get(bidder);
+
+        if (!bidderLsEntry || (bidderLsEntry && isCachedDataExpired(bidderLsEntry[lastUpdated], fetchRate || DEFAULT_FETCH_RATE_IN_DAYS))) {
+          window.fetch(`${fetchUrl}?bidder=${bidder}`, {browsingTopics: true})
```

Comment:
I have some dumb  questions about this - due to how poorly I understand topics.

I read this as "get some topics to add to the bid stream, while letting the remote end know the topics we already have in this origin".

I don't understand why the endpoint - which I believe is meant to be an SSP - needs to know the topics we see on our end, if all they're doing is telling us what we should add to the list of topics sent out later in the auction. What are they doing with them?

I'd understand a bit more if this change was about enabling the topics header on requests sent out by bidders - although even there, how is better than populating ORTB, which we are doing already?

Reply:
It's maybe because the topics one can see on the page depends on the origin of the caller?

All partners in the same page would not necessarily see the same topics for the same auction (what I mean is that, for the same browser, on the same page, at the same moment, the browser will not respond with the same topics API depending on the origin (eTLD+1) of the call.

So, every bidder has some interest to fetch the topics observed by their own origin domain (eTLD+1).

Reply:
> It's maybe because the topics one can see on the page depends on the origin of the caller?

This may make sense for iframes (if the iframes we are doing make sense - aren't they just a way to circumvent this "in-browser only" paradigm?), but the new call in this PR is just hitting a JSON endpoint, there's no page with its own topics that may be affected by this.

Reply:
@dgirardi please see https://developer.chrome.com/en/docs/privacy-sandbox/topics/#use-headers-to-access-and-observe-topics and https://developer.chrome.com/docs/privacy-sandbox/topics/demo/#the-topics-api-headers-demo

In order for the browser fetch to include the topics observed over the user lifetime, the fetch must be made in this way. The lifetime topics for the users observable to that endpoint are included, not the topic of the current request. The module assumption is that the endpoint will simply reflect that header back to the user as a json.
![image](https://github.com/prebid/Prebid.js/assets/1683175/03f116e8-17a1-438b-b534-39a3cf845a42)